src/ferc_xbrl_extractor/cli.py

```diff
@@ -99,11 +106,38 @@ def parse():
     return parser.parse_args()
 
 
+def load_extracted(
+    extracted: xbrl.ExtractOutput,
+    sqlite_uri: str,
+    duckdb_path: str | None,
+    clobber: bool,
+) -> None:
+    """Write extracted data to SQLite/Duckdb databases."""
+    engine = create_engine(sqlite_uri)
+    if clobber:
+        helpers.drop_tables(engine)
+
+    for table_name, data in extracted.table_data.items():
+        # Loop through tables and write to database
+        if not data.empty:
+            # Write to SQLite
+            with engine.begin() as sqlite_conn:
+                data.to_sql(table_name, sqlite_conn, if_exists="append")
+
+            # Write to duckdb if passed a path to duckdb
+            if duckdb_path is not None:
+                with duckdb.connect(duckdb_path) as duckdb_conn:
+                    duckdb_conn.execute(
+                        f"CREATE TABLE {table_name} AS SELECT * FROM data"  # noqa: S608
```

Comment:
Is there a reason that the DuckDB version doesn't need the same check for an existing table, and the option to append if it exists that the SQLite output has?

Reply:
It doesn't seem like we would ever expect to write to a table twice in one run, and also I don't think I've ever run this without `--clobber` (and then not immediately re-run with `--clobber`)... Maybe we should remove the clobber flag and just run `data.to_sql(..., if_exists="replace")` for the SQLite and `CREATE TABLE OR REPLACE` for the DuckDB?